bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/LedgerDirsMonitor.java

```diff
@@ -64,11 +64,21 @@ public LedgerDirsMonitor(final ServerConfiguration conf,
         this.minUsableSizeForHighPriorityWrites = conf.getMinUsableSizeForHighPriorityWrites();
         this.conf = conf;
         this.diskChecker = diskChecker;
-        this.diskUsages = ldm.getDiskUsages();
-        this.ldm = ldm;
+        this.ledgerDirsManagers = Collections.singletonList(ldm);
```

Comment:
instead of assigning variables here, you should just call other constructor

Reply:
also why do you need this constructor?

Bookie is the only (productcode) class which creates LedgerDirsMonitor and it uses the other constructor. So I dont see why this constructor should exist. I dont think it is valid to Keep it just for the test code .

Reply:
Make sense! 👍